pkg/ide/terminal.go

```diff
@@ -4,22 +4,28 @@
 package ide
 
 import (
+	"fmt"
 	"os"
 	"os/exec"
+	"strings"
 
 	"github.com/daytonaio/daytona/cmd/daytona/config"
 )
 
-func OpenTerminalSsh(activeProfile config.Profile, workspaceId string, projectName string, gpgKey string, args ...string) error {
-	err := config.EnsureSshConfigEntryAdded(activeProfile.Id, workspaceId, projectName, gpgKey)
+func OpenTerminalSsh(activeProfile config.Profile, workspaceId string, projectName string, gpgKey string, sshOptions []string, args ...string) error {
+
```

Comment:
Let's remove the extra newline here and we're good to go.

Reply:
done